benchmark/benchmark.md

```diff
@@ -0,0 +1,34 @@
+# Benchmarking
+
+## Quick Start
+
+Benchmarking runs thru the `benchmark.sh` bash script.
+
+Comparing two branches online run 
+
+    cd /path/to/benchmark/directory/
+    sh benchmark.sh -b test-branch-one -m test-branch-two
+
+the github repositiory action will automatically run 
+
+    sh benchmark.sh -m main -b the-branch-for-the-pr
+
+if you wish to run this test against your local repository vs head/main
+
+    sh benchmarck.sh -l true (or any value) 
+
+Additionally if you want to run the larger corpus for any of these
+tests you can run the flag `-s` (size)
+    
+    sh benchmarck.sh -l true -s true
```

Comment:
These funky `true` values are another strike against bash. I think we could do these better, but we lack the expertise. :/

Reply:
yeah lets make this better